pandas_gbq/gbq.py

```diff
@@ -190,18 +194,23 @@ class TableCreationError(ValueError):
 class GbqConnector(object):
     scope = 'https://www.googleapis.com/auth/bigquery'
 
-    def __init__(self, project_id, reauth=False, verbose=False,
+    def __init__(self, project_id, reauth=False, verbose=None,
```

Comment:
Can we remove the verbose parameter here since we don't support using `GbqConnector` directly (it's not documented)?